PanoptesNetClient/PanoptesNetClient/IRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json.Linq;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+
+namespace PanoptesNetClient
+{
+    public interface IRequest
```

Comment:
This name is kind of confusing. I'm not really sure what the I stands for. May be better to name this something like `PanoptesRequest`

Reply:
The "I" is a C# naming convention for interface. It could be renamed, but to follow the convention should probably be `IPanoptesRequest`.